docs/Dockerfile

```diff
@@ -7,11 +7,13 @@ COPY . /src
 # Reset the /docs dir so we can replace the theme meta with the new repo's git info
 RUN git reset --hard
 
+RUN grep "VERSION =" /src/version/version.go  | sed 's/.*"\(.*\)".*/\1/' > /docs/VERSION
```

Comment:
This isn't the actual version number. Get the version number through the git tag.